supersuit/utils/agent_indicator.py

```diff
@@ -77,13 +77,22 @@ def change_observation(obs, space, indicator_data):
         if ndims == 1:
             old_len = len(obs)
             new_obs = np.pad(obs, (0, num_indicators))
-            new_obs[indicator_num + old_len] = np.max(space.high)
+            # if all spaces are finite, use the max, otherwise use 1.0 as agent indicator
+            if not np.isinf(space.high).any():
+                new_obs[indicator_num + old_len] = np.max(space.high)
+            else:
+                new_obs[indicator_num + old_len] = 1.0
+
             return new_obs
         elif ndims == 3 or ndims == 2:
             obs = obs if ndims == 3 else np.expand_dims(obs, 2)
             old_shaped3 = obs.shape[2]
             new_obs = np.pad(obs, [(0, 0), (0, 0), (0, num_indicators)])
-            new_obs[:, :, old_shaped3 + indicator_num] = np.min(space.high)
```

Comment:
My only question @KaleabTessera is why was the code previously `np.min(space.high)` whereas now it is `np.max(space.high)`, would this not mean different behavior?

Reply:
I'm going to trust your tests ensure this behavior is correct and make a release because I need to today for something else, if this ends up being a bug we can do a hotfix release fixing it though

Reply:
Have this same question too, would like a clarification.